server/views/pages/prisonerReport.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "../partials/layout.njk" %}
+{% from "../partials/breadCrumb.njk" import breadcrumb %}
+{% from "../macros/incidentDetailsSummaryList.njk" import incidentDetailsSummaryList %}
+{% from "../macros/printLink.njk" import printLink %}
+{% from "../partials/prisonerPlaybackSection.njk" import prisonerPlaybackSection %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% block pageTitle %}
+	Prisoner’s report
+{% endblock %}
+
+{% block beforeContent %}
+ {{ breadcrumb() }}
+{% endblock %}
+
+{% block content %}
+
+ <h1 class="govuk-heading-l">{{ prisoner.friendlyName | possessive }} report</h1>
+
+ <div class="prisonerPlaybackSection">
+  {{ prisonerPlaybackSection(prisoner.displayName, prisoner.prisonerNumber, prisoner.currentLocation) }}
+ </div>
+
+ <div class="prisoner-report-top-bar govuk-!-margin-top-5">
+  <p class="govuk-body"><span class="govuk-!-font-weight-bold">Report number: </span>{{ data.reportNo }}</p>
+  {{ printLink('Print a copy of this report', 'right', 'govuk-!-padding-right-0') }}
+ </div>
+
+ <div class="govuk-!-margin-top-5">
```

Comment:
How similar is all this to `checkYourAnswers.njk`? Is it a strong candidate for a macro?

Reply:
For some reason Eddie has changed where the 'change' links are so some of the CSS is quite different

Reply:
But i was going to macro some of it for the other prisoner-report page when I started that

Reply:
Oh yeah - surely that's an error on his behalf? Can we check, because it seems unnecessary to have 2 versions of what is effectively the same data just for Change link positioning?!

If he doesn't agree, we'll make hime!